.github/scripts/latest_version_release.sh

```diff
@@ -28,47 +26,40 @@ pushd "$GITHUB_WORKSPACE/charts/$chart_dir"
     new_chart_version=$new_version && echo "The new Chart version is: ${new_chart_version}"
-    sed -i "s/version:.*/version: ${new_chart_version}/g" Chart.yaml
+    sed  -i "s/version:.*/version: ${new_chart_version}/g" Chart.yaml
 
     # edit app version
-    sed -i "s/appVersion.*/appVersion: ${app_version}/g" Chart.yaml
-    
-    if [[ -z "${sra_chart}" ]]; then
-        git add -A && git commit -m "Updated ${service} helm chart version to latest: ${app_version}" || die "Failed to commit changes to git"
-        git push origin HEAD
-    fi
-popd
+    if [[ "${chart_dir}" == "akeyless-secure-remote-access" ]]; then
+      # edit app service version
+      if [[ "${service}" == "ssh-proxy" ]]; then
+          sra_inner_chart="sshVersion"
+      elif [[ "${service}" == "zero-trust-bastion" ]]; then
+          sra_inner_chart="ztbVersion"
+      else
+          die "Bad SRA service name"
+      fi
+
+      sed -i "s/${sra_inner_chart}.*/${sra_inner_chart}: ${app_version}/g" Chart.yaml
+      # edit sra app version
+      ztb_app_ver=$(grep 'ztbVersion' Chart.yaml | awk '{print $2}')
+      ssh_app_ver=$(grep 'sshVersion' Chart.yaml | awk '{print $2}')
+      sed -i "s/appVersion.*/appVersion: ${ztb_app_ver}_${ssh_app_ver}/g" Chart.yaml
 
-if [[ -n "${sra_chart}" ]]; then
-    pushd "$GITHUB_WORKSPACE/charts/$sra_chart"
-        # bump chart version
-        chart_version=$(grep '^version:[[:space:]][[:digit:]]' Chart.yaml | awk '{print $2}') || die "Failed to retrieve current chart version"
-        bump_version "${chart_version}" "${major_minor_patch}"
-        new_chart_version=$new_version && echo "The new Chart version is: ${new_chart_version}"
-        sed -i "s/version:.*/version: ${new_chart_version}/g" Chart.yaml
+      git add -A && git commit -m "Updated ${service} helm chart version to latest: ${app_version}" || die "Failed to commit changes to git"
+      git push origin HEAD
 
-        # edit app service version
-        if [[ "${service}" == "ssh-proxy" ]]; then
-            sra_inner_chart="sshVersion"
-        elif [[ "${service}" == "zero-trust-bastion" ]]; then
-            sra_inner_chart="ztbVersion"
-        else
-            die "Bad SRA service name"
-        fi
+      echo "$chart_dir - ${service} app version was successfully updated to latest: ${app_version}"
+      echo "$chart_dir Helm chart version was updated to: ${new_chart_version}"
 
-        sed -i "s/${sra_inner_chart}.*/${sra_inner_chart}: ${app_version}/g" Chart.yaml
-        # edit app version
-        ztb_app_ver=$(grep appVersion ../akeyless-zero-trust-bastion/Chart.yaml | awk '{print $2}')
-        ssh_app_ver=$(grep appVersion ../akeyless-ssh-bastion/Chart.yaml | awk '{print $2}')
-        sed -i "s/appVersion.*/appVersion: ${ztb_app_ver}_${ssh_app_ver}/g" Chart.yaml
-        
+    else
+        sed -i "s/appVersion.*/appVersion: ${app_version}/g" Chart.yaml
+    fi
+
+    if [[ "${chart_dir}" != "akeyless-secure-remote-access"  ]]; then
```

Comment:
Why? you can use the else statement on line 54 to do the git add and push since you already know its not "akeyless-secure-remote-access" dir

Reply:
updated